pdf/core/encoding.go

```
@@ -1335,6 +1481,110 @@ func (this *RawEncoder) EncodeBytes(data []byte) ([]byte, error) {
 }
 
 //
+// CCITTFax encoder/decoder (dummy, for now)
+//
+type CCITTFaxEncoder struct{}
+
+func NewCCITTFaxEncoder() *CCITTFaxEncoder {
+	return &CCITTFaxEncoder{}
+}
+
+func (this *CCITTFaxEncoder) GetFilterName() string {
+	return StreamEncodingFilterNameCCITTFax
+}
+
+func (this *CCITTFaxEncoder) MakeDecodeParams() PdfObject {
+	return nil
+}
+
+// Make a new instance of an encoding dictionary for a stream object.
+func (this *CCITTFaxEncoder) MakeStreamDict() *PdfObjectDictionary {
+	return MakeDict()
+}
+
+func (this *CCITTFaxEncoder) DecodeBytes(encoded []byte) ([]byte, error) {
+	return encoded, nil
```

Should return an error (unsupported)